test/batchTraverse.ts

```diff
@@ -19,7 +19,7 @@ describe('batchTraverse', () => {
       )
     )()
     assert.deepStrictEqual(actual, E.right([1, 2, 3, 4]))
-    assert.deepStrictEqual(log, ['Executing 1', 'Executing 2', 'Executing 3', 'Executing 4'])
+    assert.deepStrictEqual(log, ['Executing 2', 'Executing 1', 'Executing 4', 'Executing 3'])
```

Comment:
I'm not sure why the order has changed in `log` but not `actual`...

Reply:
My understanding is that the result will have a consistent order, but the actual order of operations won't (hence the logging appearing an unexpected order). The `log` assertion shouldn't be checking the order, so a `sort` will get round this.

Reply:
`sort` added in https://github.com/gcanti/fp-ts-contrib/pull/89/commits/6ba8abd738f11d70047c773c63b39de17a6951af